includes/class-amp-story-post-type.php

```diff
@@ -1500,4 +1512,26 @@ public static function change_embed_iframe_attributes( $output, $post ) {
 			$output
 		);
 	}
+
+	/**
+	 * Adds a new max image size to the images sizes available in the Media Library.
+	 *
+	 * In the AMP story editor, when selecting Background Media,
+	 * it will use this custom image size.
+	 *
+	 * @param array $image_sizes {
+	 *     An associative array of image sizes.
+	 *
+	 *     @type string $slug Image size slug, like 'medium'.
+	 *     @type string $name Image size name, like 'Medium'.
+	 * }
+	 * @return array $image_sizes The filtered image sizes.
+	 */
+	public static function add_new_max_image_size( $image_sizes ) {
+		if ( self::POST_TYPE_SLUG === get_post_type() ) {
+			$image_sizes[ self::STORY_PAGE_IMAGE_SIZE ] = __( 'AMP Story Maximum Image', 'amp' );
```

Comment:
How about just “Story Background” or “Story Page”. Should minimize showing “AMP” to the user.

Reply:
Good idea, https://github.com/ampproject/amp-wp/pull/2509/commits/25d105733f92d946561c8e82621f6e37f0d69d52 changes the name to `'Story Background'` if that's alright.